src/js/components/TextInput/TextInput.js

```diff
@@ -393,7 +393,7 @@ const TextInput = forwardRef(
           }
           onTab={showDrop ? closeDrop : undefined}
           onUp={
-            showDrop &&
+            !showDrop &&
```

Comment:
I think this is the key, that we don't process "Up" with both Keyboard listeners. Am I right?

Reply:
Exactly.